internal/host/refresh_status_preprocessor.go

```diff
@@ -72,11 +73,19 @@ func (r *refreshPreprocessor) preprocess(c *validationContext) (map[validationID
 			Status:  status,
 			Message: strings.Join(result.Reasons, "\n"),
 		})
+		sortByValidationResultID(validationsOutput[category])
```

Comment:
Can yo please add a test that validate that it called the same as you did in cluster_test

Reply:
Done